ymap/ymapexport.py

```diff
@@ -164,61 +164,72 @@ def ymap_from_object(obj):
     for child in obj.children:
         # Entities
         if export_settings.ymap_exclude_entities == False and child.sollum_type == SollumType.YMAP_ENTITY_GROUP:
-            for entity in child.children:
-                if entity.sollum_type == SollumType.DRAWABLE:
-                    ymap.entities.append(entity_from_obj(entity))
+            for entity_obj in child.children:
+                if entity_obj.sollum_type == SollumType.DRAWABLE:
+                    ymap.entities.append(entity_from_obj(entity_obj))
                 else:
                     logger.warning(
-                        f"Object {entity.name} will be skipped because it is not a {SOLLUMZ_UI_NAMES[SollumType.DRAWABLE]} type.")
+                        f"Object {entity_obj.name} will be skipped because it is not a {SOLLUMZ_UI_NAMES[SollumType.DRAWABLE]} type.")
 
         # Box occluders
         if export_settings.ymap_box_occluders == False and child.sollum_type == SollumType.YMAP_BOX_OCCLUDER_GROUP:
             obj.ymap_properties.content_flags_toggle.has_occl = True
 
-            for cargen in child.children:
-                if cargen.sollum_type == SollumType.YMAP_BOX_OCCLUDER:
-                    ymap.box_occluders.append(box_from_obj(cargen))
-                    calculate_extents(ymap, cargen)
+            for box_obj in child.children:
+                rotation = box_obj.rotation_euler
+                if rotation.x > 0.01 or rotation.y > 0.01:
+                    logger.error(
+                        f"Box occluders only support Z-axis rotation. Skipping {box_obj.name} due to X/Y rotation.")
+                    continue
```

Comment:
`rotation_euler` can be negative. Should be wrapped it in `abs()`
Also, in other places where we skip objects, `logger.warning` is used instead of `logger.error`. I think it should be a warning here as well for consistency.
```suggestion
                if abs(rotation.x) > 0.01 or abs(rotation.y) > 0.01:
                    logger.warning(
                        f"Box occluders only support Z-axis rotation. Skipping {box_obj.name} due to X/Y rotation.")
                    continue
```

Reply:
Oh nice.
I kept the error tho, otherwise the user can't see what is wrong since no popup will throw, and the "Exported in X seconds" message appears.

![image](https://github.com/Skylumz/Sollumz/assets/47056777/1a869081-5916-42ae-86d4-fdc86ef53af1)